atlasdb-impl-shared/src/main/metrics/targeted-sweep.yml

```diff
@@ -5,6 +5,37 @@ namespaces:
   targetedSweepProgress:
     docs: Metrics for tracking the progress of Targeted Sweep.
     metrics:
+      closedBucketIdentifier:
+        type: gauge
+        docs: The bucket identifier that was associated with the last closed bucket.
+      bucketAssignerStatus:
+        type: gauge
+        docs: | 
+          The result of the bucket assigner's last run as an integer.
+          0. INCOMPLETE
+          1. CLOSED
+          2. CONTENTION_ON_WRITES
+          
+          INCOMPLETE represents the fact that the bucket assigner ran but
+          could not close the bucket. CLOSED represents the fact that the bucket assigner closed the bucket.
+          CONTENTION_ON_WRITES is when the bucket assigner may be racing against another bucket assigner on a different
+          node and failed to complete.
+          A healthy state is when the bucket assigner is running regularly, and periodically emitting a CLOSED status.
```

Comment:
here, 'running regularly' means that you're getting `INCOMPLETE` (and, as the PR as written says, with the odd `CLOSED`.)